R/utilities.R

```diff
@@ -773,9 +773,17 @@ vec_rbind0 <- function(..., .error_call = current_env(), .call = caller_env()) {
   )
 }
 
+# This function is used to vectorise the following pattern:
+#
+# list$name1 <- list$name1 %||% value
+# list$name2 <- list$name2 %||% value
+#
+# and express this pattern as:
+#
+# replace_null(list, name1 = value, name2 = value)
 replace_null <- function(list, ..., env = caller_env()) {
```

Comment:
Maybe `list` is not the best argument name as it masks a very common/basic function name

Reply:
Agreed